pkgs/development/python-modules/django-anymail/default.nix

```diff
@@ -23,28 +24,27 @@ buildPythonPackage rec {
     hash = "sha256-unmbYQFLeqfqE1uFLMPLUad1UqA+sgbTzwRfpRhM3ik=";
   };
 
+  nativeBuildInputs = [
+    hatchling
+  ];
+
   propagatedBuildInputs = [
-    six
     requests
     django
-    boto3
   ];
 
   nativeCheckInputs = [
-    pytestCheckHook
-    pytest-django
     mock
-  ];
+    responses
+  ] ++ passthru.optional-dependencies.amazon-ses;
 
-  disabledTests = [
-    # Require networking
-    "test_debug_logging"
-    "test_no_debug_logging"
-  ];
+  passthru.optional-dependencies = {
+    amazon-ses = [ boto3 ];
+  };
 
-  pythonImportsCheck = [ "anymail" ];
+  checkPhase = "CONTINUOUS_INTEGRATION=1 python runtests.py";
```

Comment:
```suggestion
  checkPhase = ''
    runHook preCheck
    CONTINUOUS_INTEGRATION=1 python runtests.py
    runHook postCheck
  '';
```

Reply:
Did the package stop using pytest?

Reply:
I checked the history and I couldn’t find any trace of it ever using pytest. 🤷